src/Repository/PostsRepository.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Repository;
+
+use App\Library\PDO;
+use App\Entity\PostsEntity;
+
+class PostsRepository
+{
+	//Post.php:35.d
+	public static function persistEntity(PostsEntity $entity)
+	{
+		$insertPost = PDO::instance()->prepare("INSERT INTO posts VALUES(?, ?, ?, ?, ?, ?)");
+		$insertPost->execute($entity->toArray());
+	}
+	//Post.php:59.d
+	public static function getPostByNotDeleted()
+	{
+		$getPosts = PDO::instance()->prepare("SELECT * FROM posts WHERE deleted=? ORDER BY id DESC");
+		$getPosts->execute(['no']);
+		while ($postRow = $getPosts->fetch())
+		yield new PostsEntity(...$postRow);
+	}
+	//Post.php:59.d
+	public static function getRowPostByNotDeleted(mixed $deleted)
```

Comment:
Try to avoid `mixed` type and be more strict.

Also name indicates you want to have collection of not deleted posts, so why to parametrise it?

What is more, you are missing return type.